corebehrt/azure/util.py

```
@@ -46,13 +56,40 @@
     return MLClient.from_config(DefaultAzureCredential())
 
 
+def create_job(
+    name: str,
+    config: dict,
+    compute: str,
+    register_output: dict = dict(),
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)